frontend/src/pages/blog/CreateArticle.js

```diff
@@ -51,15 +52,19 @@ const CreateArticle = () => {
 
   const handleFileChange = (event) => {
     const file = event.target.files[0];
-    console.log(file);
-    console.log(file.text());
-    file.text().then((t) => {
-      const lines = t.split("\n");
-      console.log(lines);
-      setTitle(lines[0]);
-      setSummary(lines[1]);
-      setBody(lines.slice(3).join("\n"));
-    });
+    console.log(file.size)
+    console.log(Config.FILE_SIZE_MAX)
+    if (file.size < Config.FILE_SIZE_MAX) {
+        file.text().then((t) => {
+          const lines = t.split("\n");
+          setTitle(lines[0]);
+          setSummary(lines[1]);
+          setBody(lines.slice(3).join("\n"));
+        });
+    }
+    else {
+        setError("File is too large!")
```

Comment:
Mention the max file size (ie `Config.FILE_SIZE_MAX` as a string) in the error message for user-friendliness

Reply:
Yes that's a good idea